example/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-alpine
+
+# Copy the worker
+COPY src /opt/opencti-connector-example
```

Comment:
Moving the `COPY` at the end avoids invalidating the cache of the intermediate layers if the connector source code change. Example:
```dockerfile
FROM python:3.8-alpine

WORKDIR  /opt/opencti-connector-example

COPY src/requirements.txt  ./

RUN apk --no-cache add build-base && \
    pip3 install --no-cache-dir -r requirements.txt && \
    apk del build-base

COPY src entrypoint.sh ./
 
ENTRYPOINT ["/bin/sh", "-c", "/opt/opencti-connector-example/entrypoint.sh"]

```